lib/database/models/resto.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:objectbox/objectbox.dart';
+
+import 'promo.dart' show Promo;
+import 'user.dart' show User;
+
+@Entity()
+class Resto {
+  @Id()
+  int id = 0;
+
+  @Backlink()
+  final followerUsers = ToMany<User>();
```

Comment:
Maybe leaving just "followers" is ok.

Reply:
I am not sure why we want to persist a relation between users and restos but I think one of bot "backlinks" is not needed. By following this [doc](https://docs.objectbox.io/relations#access-many-to-many-in-the-reverse-direction) I think one of them can be removed.

Reply:
just `followers` sounds good, I had read the doc at that point, but at the time of coding I felt that was more explicit declaring in both places, maybe is an obsession of mine to explicitly put in all places the things jajaja. I'm gonna change because this surely behave wrong